dev-support/test-patch.sh

```diff
@@ -2611,8 +2392,8 @@ function check_patch_javac
           "${PATCH_DIR}/patch-javac-${fn}-warning.txt"
       fi
 
-      numbranch=$(count_javac_probs "${PATCH_DIR}/branch-javac-${fn}-warning.txt")
-      numpatch=$(count_javac_probs "${PATCH_DIR}/patch-javac-${fn}-warning.txt")
+      numbranch=$(${BUILDTOOL}_count_javac_probs "${PATCH_DIR}/branch-javac-${fn}-warning.txt")
+      numpatch=$(${BUILDTOOL}_count_javac_probs "${PATCH_DIR}/patch-javac-${fn}-warning.txt")
```

Comment:
shellcheck: 19: note: Double quote to prevent globbing and word splitting. [SC2086]